src/client/dfs/dfs.c

```diff
@@ -1613,6 +1607,99 @@ dfs_readdir(dfs_t *dfs, dfs_obj_t *obj, daos_anchor_t *anchor, uint32_t *nr,
 	return rc;
 }
 
+int
+dfs_lookup_rel(dfs_t *dfs, dfs_obj_t *parent, const char *name, int flags,
+	       dfs_obj_t **_obj, mode_t *mode)
+{
+	dfs_obj_t		*obj;
+	struct dfs_entry	entry = {0};
+	bool			exists;
+	int			daos_mode;
+	int			rc = 0;
+
+	if (dfs == NULL || !dfs->mounted)
+		return -DER_INVAL;
+	if (_obj == NULL)
+		return -DER_INVAL;
+	if (parent == NULL)
+		parent = &dfs->root;
+	else if (!S_ISDIR(parent->mode))
+		return -DER_NOTDIR;
+
+	rc = check_name(name);
+	if (rc) {
+		D_ERROR("Invalid file/dir Name\n");
+		return rc;
+	}
+	rc = check_access(dfs, geteuid(), getegid(), parent->mode, X_OK);
+	if (rc) {
+		D_ERROR("Permission Denied.\n");
+		return rc;
+	}
+
+	daos_mode = get_daos_obj_mode(flags);
+	if (daos_mode == -1) {
+		D_ERROR("Invalid access mode.\n");
+		return -DER_INVAL;
+	}
+
+	D_ALLOC_PTR(obj);
+	if (obj == NULL)
+		return -DER_NOMEM;
+
+	rc = fetch_entry(parent->oh, DAOS_TX_NONE, name, true, &exists,
+			 &entry);
+	if (rc)
+		D_GOTO(err_obj, rc);
+
+	if (!exists) {
+		D_ERROR("Entry %s does not exist\n", name);
```

Comment:
This is a perfectly normal thing to do in POSIX, so DAOS should not log this as D_ERROR().

Reply:
yes you're right, i think i copied this from somewhere that i need to fix there too.
will fix